src/blockchain/log-processors/order-created.ts

```diff
@@ -37,55 +21,55 @@ interface OrderCreatedOnContractData {
 }
 
 export function processOrderCreatedLog(db: Knex, augur: Augur, trx: Knex.Transaction, log: FormattedEventLog, callback: ErrorCallback): void {
-  trx.select(["marketID", "outcome"]).from("tokens").where({ contractAddress: log.shareToken }).asCallback((err: Error|null, tokensRows?: Array<TokensRow>): void => {
+  const amount: string = log.amount;
+  const price: string = log.price;
+  const orderType: string = log.orderType;
+  const moneyEscrowed: string = log.moneyEscrowed;
+  const sharesEscrowed: string = log.sharesEscrowed;
+  const shareToken: Address = log.shareToken;
+  trx.first("marketID", "outcome").from("tokens").where({ contractAddress: shareToken }).asCallback((err: Error|null, tokensRow?: TokensRow): void => {
     if (err) return callback(err);
-    if (!tokensRows || !tokensRows.length) return callback(new Error("market and outcome not found"));
-    const { marketID, outcome } = tokensRows[0];
-    trx.select(["minPrice", "maxPrice", "numTicks"]).from("markets").where({ marketID }).asCallback((err: Error|null, marketsRows?: Array<MarketsRow>): void => {
+    if (!tokensRow) return callback(new Error("market and outcome not found"));
+    const marketID = tokensRow.marketID!;
+    const outcome = tokensRow.outcome!;
+    trx.first("minPrice", "maxPrice", "numTicks").from("markets").where({ marketID }).asCallback((err: Error|null, marketsRow?: MarketsRow): void => {
       if (err) return callback(err);
-      if (!marketsRows || !marketsRows.length) return callback(new Error("market min price, max price, and/or num ticks not found"));
-      const { minPrice, maxPrice, numTicks } = marketsRows[0];
+      if (!marketsRow) return callback(new Error("market min price, max price, and/or num ticks not found"));
+      const minPrice = marketsRow.minPrice!;
+      const maxPrice = marketsRow.maxPrice!;
+      const numTicks = marketsRow.numTicks!;
       const ordersPayload = { _orderId: log.orderId };
-      parallel({
-        orderType: (next: AsyncCallback): void => augur.api.Orders.getOrderType(ordersPayload, next),
-        price: (next: AsyncCallback): void => augur.api.Orders.getPrice(ordersPayload, next),
-        amount: (next: AsyncCallback): void => augur.api.Orders.getAmount(ordersPayload, next),
-        sharesEscrowed: (next: AsyncCallback): void => augur.api.Orders.getOrderSharesEscrowed(ordersPayload, next),
-        moneyEscrowed: (next: AsyncCallback): void => augur.api.Orders.getOrderMoneyEscrowed(ordersPayload, next),
-      }, (err: Error|null, onContractData: OrderCreatedOnContractData): void => {
+      const fullPrecisionAmount = convertFixedPointToDecimal(amount, WEI_PER_ETHER);
+      const fullPrecisionPrice = denormalizePrice(minPrice, maxPrice, convertFixedPointToDecimal(price, numTicks));
+      if (new BigNumber(fullPrecisionPrice, 10).eq(new BigNumber(-10, 10))) process.exit();
```

Comment:
?

Reply:
Oops, I put that in as a debugging check and forgot to remove it... yikes :grimacing: